scripts/install-cni.sh

```diff
@@ -27,8 +27,19 @@ fi
 if [ -w /host/etc/cni/net.d ]; then
   cni_spec=$(echo ${CNI_SPEC_TEMPLATE:-} | sed -e "s#podCidr#${pod_cidr:-}#g")
 
+  node_ipv6_addr=$(curl -s -k --fail "http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/ipv6s" -H "Metadata-Flavor: Google" | jq '.[0]') ||:
```

Comment:
Should we fetch IPv6 conditioned on ENABLE_PRIVATE_IPV6_ACCESS?

It is possible the subnet is enabled for private IPv6 access but user does not enable private IPv6 access for the K8s cluster.

Reply:
Done.